content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -4,247 +4,206 @@ reviewers:
-  * `--authorization-mode=AlwaysAllow` This flag allows all requests. Use this flag only if you do not require authorization for your API requests.
+Kubernetes checks authorization modules based on the order that you specify them
+on the API server's command line, so an earlier module has higher priority to allow
+or deny a request.
 
-You can choose more than one authorization module. Modules are checked in order
-so an earlier module has higher priority to allow or deny a request.
+For more information on command line arguments to the API server, read the
+[`kube-apiserver` reference](/docs/reference/command-line-tools-reference/kube-apiserver/).
 
-## Configuring the API Server using an Authorization Config File
+### Configuring the API Server using an authorization config file
 
 {{< feature-state state="alpha" for_k8s_version="v1.29" >}}
 
-The Kubernetes API server's authorizer chain can be configured using a
-configuration file.
+In order to customise the authorizer chain using a configuration file,
+you need to enable the `StructuredAuthorizationConfiguration`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
 
 You specify the path to that authorization configuration using the
-`--authorization-config` command line argument. This feature enables
-creation of authorization chains with multiple webhooks with well-defined
-parameters that validate requests in a certain order and enables fine grained
-control - such as explicit Deny on failures. An example configuration with
-all possible values is provided below.
+`--authorization-config` command line argument. Specifying configuration
+via a local file allows you to define authorization chains with multiple
+webhooks. Authorization chain elements can have well-defined parameters
+that validate requests in a certain order; this enables fine grained
+control - such as explicit Deny on failures.
 
-In order to customise the authorizer chain, you need to enable the
-`StructuredAuthorizationConfiguration` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+Setting both `--authorization-config` path and configuring an authorization
+webhook using the `--authorization-mode` and `--authorization-webhook-*`
+command line arguments is not allowed. If you try this, there will be an error
+on startup, and the API server does not run.
 
-Note: When the feature is enabled, setting both `--authorization-config` and
-configuring an authorization webhook using the `--authorization-mode` and
-`--authorization-webhook-*` command line flags is not allowed. If done, there
-will be an error and API Server would exit right away.
+{{< note >}}
+When configuring the authorizer chain using a configuration file, make sure all the
+API server replicas have the same configuration file contents available locally.
 
-{{< caution >}}
-While the feature is in Alpha/Beta, there is no change if you want to keep on
```

Comment:
Can you please rebase on top of the current? I pusher https://github.com/kubernetes/website/pull/45138 with verbiage about the command line will continue to work for all future k8s versions

Reply:
Will do.

Reply:
Same with text about upgrade and taking a backup. we should keep that. PTAL https://github.com/kubernetes/website/pull/45138

Reply:
_This_ PR isn't the place to merge with https://github.com/kubernetes/website/pull/45138

What I can do instead: help with combining these changes into _dev-1.30_.

We can also not change this page until the v1.30 release, and then rebase this PR on top of _main_. I'd prefer not to do that, but we can.

Reply:
If this PR is meant for 1.29 updates, then I think there is value in adding. because we did get questions about if command line flags will go away in the future:
> If you want to keep using command line flags to configure authorization instead of a configuration file, those will continue to work as-is.

Reply:
I'll reword that, but yes, let's make that point.

Reply:
v1.30 is released now, and I've rebased; also I reworded.